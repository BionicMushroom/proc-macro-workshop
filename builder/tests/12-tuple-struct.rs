use derive_builder::Builder;

#[derive(Builder)]
struct Test(i32, u32, String, Option<String>, Vec<String>);

fn main() {}
