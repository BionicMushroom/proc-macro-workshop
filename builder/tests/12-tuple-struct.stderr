error: expected struct with named fields, found tuple-struct
 --> tests/12-tuple-struct.rs:4:12
  |
4 | struct Test(i32, u32, String, Option<String>, Vec<String>);
  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
